app-web/plugins/gatsby-source-github-all/utils/helpers.js

```diff
@@ -369,9 +369,23 @@ const assignPositionToResource = source => (resource, position) => ({
   },
 });
 
+/**
+ * creates a string that is usable in a natural sort
+ * it padds numbers with zeros so that it will sort normally
+ * ie [0, 0, 1], [0, 0, 10] => 0000.0000.0001 , 0000.0000.0010
+ * @param {Array} pos the position array
+ */
+const convertPositionToSortableString = (maxPadding, pos) =>
```

Comment:
maybe consider hex value? aim at bigger than 9999 ;P

Reply:
Thats a really good point. I won't change it for now though. I have a feeling we will be changing how positions are maintained with the upcoming graph structure. which would make this function useless at that point:P